src/GameManager.cpp

```diff
@@ -34,3 +34,22 @@ void GameManager::printInfoToConsole(float currentFPS) {
 	std::cout << " Number of dynamic objects in world: " << currentWorld_->getNumDynamicGameObjects();
 	std::cout << " Number of dynamic objects hit: " << currentWorld_->getNumBunniesHit() << '\r' <<  std::flush;
 }
+
+void GameManager::reportScore(float score) {
+	score_ += score;
+    std::cout << "Score: " << score_ << " (+" << score << ")" << std::endl;

```

Comment:
Not needed for this PR, but we will want this to be part of that every iteration print statement with the FPS in the main loop I commented out. That way it just polls for the score and doesn't print a new line to the console every time.

But since that is still borked on linux I believe this is good for now.